### PR TITLE
Add tiles components and IAReactComponent from dweb-archive

### PR DESCRIPTION
**Pull request by mitra42**
> **Description**: What does this PR achieve? [feature|hotfix|refactor]

Closes #

> **Technical**: What should be noted about the implementation?

> **Testing**: Steps for reviewer to reproduce / verify this PR fixes the problem?

> **Evidence**: If this PR touches UI, please post evidence (screenshot) of it behaving correctly:

### CODE REVIEW

**Comment by mitra42** [comment]
@iisa as discussed, here's the PR for first 3 components

File: packages/ia-components/sandbox/IAReactComponent.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+const debug = require('debug')('dweb-archive:ConfigDetailsComponent');
```
Comment (iisa) [inline packages/ia-components/sandbox/IAReactComponent.js]: Suggestion: Add comment around `debug` component - and how to configure.  You mentioned that it is a current node process but there is a client side workaround to ensure that it works.

**Comment by mitra42** [comment]
Hmmm - that would put the comment on EVERY file, which seems syntactic overkill. 
The trick is: Server-side: 
`process.env.DEBUG="dweb-archive:*"` or `process.env.DEBUG="dweb-archive:ConfigDetailsComponent"` in the main JS file (e.g. in index.js if tht is what is requiring all these)

Client side it is:   `<script type="text/javascript">localStorage.debug = "dweb-archive dweb-archive:* dweb-transports dweb-transports:* dweb-objects dweb-objects:*";</script>`  or can execute that in the javascript before the `require debug` 

If neither are done, the debug doesnt slow things down , I forget if it just compiles to a noop or effectively that.

**Comment by iisa** [comment]
Ok, that is fine for now.
For the record, we want the site/components to work without localStorage overall.  We will kick that can down the road, and address it when archive.org touches the component.

**Comment by mitra42** [comment]
Understood - and note the client-side localStorage is only required to turn the debugging on, so could be easily commented out for normal use, or I'll figure out how to enable it in some other way when the need arises.

**Comment by mitra42** [comment]
I'm working on those components at the moment - getting rid of the dweb dependencies, but will do that as a separate PR

**Pull request merged**